hpc-ard-launcher-go/launcher/client.go

```diff
@@ -274,7 +274,7 @@ func (c *APIClient) prepareRequest(
 		headerParams["Content-Type"] = w.FormDataContentType()
 
 		// Set Content-Length
-		headerParams["Content-Length"] = fmt.Sprintf("%d", body.Len())
```

Comment:
This file is generated so we shouldn't be changing it. It should be excluded from linting rules.

Reply:
yeah removed -- I think this was a regex find & replace on my end